src/index.js

```diff
@@ -19,6 +20,7 @@ const commands = new Map([
 	['help', help],
 	['', help],
 	['init', init],
+	['extra', extra],
```

Comment:
It should be `extras` shouldn't it?

Reply:
It will add a single extra, so, not really.

Reply:
we ain't planning support more extras?
@jacoobes your thoughts?

Reply:
We will, but right now it is a single extra, haha.

Also, I believe creating one at a time is better since there might be conflicts with existing code and asking the user if they want to delete it or not multiple times in a command is messed up, in my opinion.

Reply:
sure